vmdk_plugin/main_linux.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
We don't need to specify explicitly the build tag if we already name the arch as the filename suffix. I borrowed this pattern from the [Docker Repo](https://github.com/moby/moby/tree/master/volume). For more information about this, refer the [Golang Doc](https://golang.org/pkg/go/build/).